<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 41, 41);</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>171</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 54, 54);</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="ButtonReset">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonRun">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Pcv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>  [g/g]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Prho">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>  [g/L]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PQt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string> [m³/h]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PDc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string notr="true">  [cm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_PDc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Dc:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Prhos">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>  [g/L]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PDu">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>  [cm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Prhos">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>ρs:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Pmu">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="placeholderText">
      <string>   [cP]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pmu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>μ:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_PDu">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Du:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Prho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>ρ:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_param">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PdP">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>  [atm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pcv">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Cv:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_PdP">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>ΔP:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_PQ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Q:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 54, 54);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Granulometric Method</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_RRB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>RRB</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_GGS">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>GGS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_best">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>best</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Sigm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Sigmoide</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Family</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Rietema">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Rietema</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_Bradley">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Bradley</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_Demco">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Demco 4H</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(181, 181, 181);</string>
     </property>
     <property name="text">
      <string>Graphics</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2D">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>2D</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_3D">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>3D</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>191</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/Hydrocyclone-fonctionnement.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuParticle_size_Distribution">
    <property name="title">
     <string>Particle-size Distribution</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuParameterization">
    <property name="title">
     <string>Parameterization</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuResults">
    <property name="title">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Information</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuOpen"/>
   <addaction name="menuSave"/>
   <addaction name="menuParticle_size_Distribution"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuParameterization"/>
   <addaction name="menuResults"/>
   <addaction name="menuInformation"/>
   <addaction name="menuReport"/>
   <addaction name="menuSettings"/>
  </widget>
 </widget>
 <resources>
  <include location="Interface/img/img.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
